Resources/Prototypes/_Pirate/Entities/Structures/Doors/Airlocks/airlocks.yml

```
@@ -6,3 +6,21 @@
     sprite: _Pirate/Structures/Doors/Airlocks/Abductor/abductor.rsi
     layoutId: AirlockAbductor
+
+- type: entity
+  parent: Airlock
+  id: AirlockInterdyne
+  suffix: Interdyne
+  components:
+  - type: Sprite
+    sprite: _Pirate/Structures/Doors/Airlocks/Standard/interdyne.rsi
+    layoutId: AirlockInterdyne
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)